### PR TITLE
fix: improve `TicketCreate` table view

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR improves the `TicketCreate` view in transaction table lists. It adds the `TicketCreate` name and adds a little bit of information (the ticket count) to the view.

I also resolved a couple of minor TS issues that popped up when I was working on this.

### Context of Change

It was an annoyance when I was working with accounts using tickets.

### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)

### TypeScript/Hooks Update

The file was already in TS.

## Before / After

image

image

## Test Plan

CI passes. I tested by hand and it worked properly.

### CODE REVIEW

**Review by JST5000 — APPROVED** [review]

File: public/locales/en-US/translations.json [hunk 1]
```diff
@@ -116,19 +116,20 @@
...
   "transaction_EscrowFinish": "Escrow Finish",
-  "transaction_OfferCancel": "Offer Cancel",
-  "transaction_OfferCreate": "Offer Create",
   "transaction_NFTokenAcceptOffer": "NFT Accept Offer",
   "transaction_NFTokenBurn": "NFT Burn",
   "transaction_NFTokenCancelOffer": "NFT Cancel Offer",
   "transaction_NFTokenCreateOffer": "NFT Create Offer",
   "transaction_NFTokenMint": "NFT Mint",
+  "transaction_OfferCancel": "Offer Cancel",
+  "transaction_OfferCreate": "Offer Create",
```
Comment (ledhed2222) [inline public/locales/en-US/translations.json:125]: are you just sorting these alphabetically?

File: src/containers/shared/components/TxDetails.tsx [hunk 2]
```diff
@@ -14,17 +14,17 @@ interface Instructions {
   cancelAfter: string;
   condition: string;
   quorum: number;
-  max: any;
+  max: number;
```
Comment (ledhed2222) [inline src/containers/shared/components/TxDetails.tsx:17]: 👍

File: public/locales/en-US/translations.json (hunk 1 above)
Comment (mvadari) [reply public/locales/en-US/translations.json:125]: yes - it was just slightly off and it annoyed me

Comment (mvadari) [reply public/locales/en-US/translations.json:125]: (I also added a `TicketCreate` translation in this file)

File: src/containers/shared/components/TxDetails.tsx [hunk 3]
```diff
@@ -384,6 +385,16 @@ const TxDetails = (props: Props) => {
     );
   }
 
+  function renderTicketCreate(): ReactElement {
+    const { t, instructions } = props;
+    const { ticketCount } = instructions;
+    return (
+      <div className="ticketCreate">
+        <span className="label">{t('ticket_count')}:</span> <span>{ticketCount}</span>
+      </div>
```
Comment (ledhed2222) [inline src/containers/shared/components/TxDetails.tsx:394]: not a big deal, but i find that when you have a literal space that you want to add to the markup it's more clear to either do:

```javascript
<span>{ }</span>
```

or

```javascript
<span>&nbsp;</span>
```

**Review by ledhed2222 — APPROVED** [review]

**Review by intelliot — APPROVED** [review]

**Pull request merged**